itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenarios.java

```diff
@@ -1607,6 +1607,40 @@ public void testTruncateWithCM() throws IOException {
   }
 
   @Test
+  public void testTruncateTempTable() throws IOException {
```

Comment:
TestReplicationScenarios.java is not an intuitive place for this test as it is not testing replication specific use case.
Looks like most of the temp table tests are under ql/src/test/queries/clientpositive/temp_*.q . Can we create a similar .q file for this testing ?